### ISSUE TITLE
Unable to build, Raspberry Pi2 Cypriot

### ISSUE THREAD

**Issue by gazzer82** [muuuh/docker-airsonos-pi#2]
I am trying to build this on a Raspberry Pi2 running the latest build of Hypriot an Docker 1.10.1 but it fails when running the NPM Install of babel@5 with a checksum failure.

Is this something you've seen? Any ideas how to get around it?

Thanks

Gareth

**Comment by muuuh** [comment]
Nope, haven't seen this error myself. I built with docker version 1.9.1, posted here, afterwards updated docker to 1.10.0, had no issues and haven't tried 1.10.1.
